python/ray/util/sgd/v2/examples/transformers/README.rst

```diff
@@ -0,0 +1,53 @@
+HuggingFace Transformers Glue Fine-tuning Example
+=================================================
+
+We've ported the ``huggingface/transformers/examples/pytorch/text-classification/run_glue_no_trainer.py`` example to
+RaySGD. This example enables fine-tuning the library models for sequence classification on the GLUE benchmark: General Language Understanding Evaluation.
+
+This script can fine-tune the following models:
+ CoLA, SST-2, MRPC, STS-B, QQP, MNLI, QNLI, RTE, WNLI.
+
+Additional information can be found at the `HuggingFace Repository
+<https://github.com/huggingface/transformers/tree/master/examples/pytorch/text-classification>`_.
+
+Single-node CPU training
+----------------------------
+
+To run an example tuning MRPC locally using CPU:
+
+.. code-block:: bash
+
+    export TASK_NAME=mrpc
+
+    python transformers_example.py \
+      --model_name_or_path bert-base-cased \
+      --task_name $TASK_NAME \
+      --max_length 128 \
+      --per_device_train_batch_size 32 \
+      --learning_rate 2e-5 \
+      --num_train_epochs 3 \
+      --output_dir /tmp/$TASK_NAME/
```

Comment:
Yeah the only concern I have here is that it could be confusing for users since it's unclear that we can still use multiple workers even if on a single node. The distinction here should be between single process vs. distributed (either on a single node or multiple nodes) as opposed to single node vs. multiple nodes. And regardless of whether it is distributed or not you can still use GPU.

Reply:
Gotcha, that makes sense. I updated this to mention that it becomes the same as running the original `run_glue_no_trainer.py` script and removed assumptions about the local environment resources.